lxd/auth/driver_tls.go

```diff
@@ -44,12 +45,15 @@ func (t *tls) CheckPermission(ctx context.Context, r *http.Request, object Objec
 		return nil
 	}
 
-	certType, isNotRestricted, projectNames, err := t.certificateDetails(details.username())
+	username := details.username()
+	id, err := t.identities.Get(api.AuthenticationMethodTLS, details.username())
 	if err != nil {
-		return err
+		return fmt.Errorf("Certificate %q missing from identity cache", username)
```

Comment:
Can we change this to be something like:

```go
return fmt.Errorf("Failed loading certificate for %q, %w", username, err)
```

And have .Get return a not found status error?